site/docs/master/contributions/ceph-radosgateway.md

```diff
@@ -0,0 +1,238 @@
+### 1. Object Store environment
```

Comment:
It'd be helpful to add a brief introduction explaining what this document covers, similar to e.g. https://github.com/vmware-tanzu/velero/blob/master/site/docs/master/contributions/oracle-config.md#introduction